rpc-interface/src/blockchain.rs

```diff
@@ -97,9 +97,6 @@ pub trait BlockchainInterface {
         &mut self,
     ) -> RPCResult<SlashedSlots, BlockchainState, Self::Error>;
 
-    async fn get_parked_validators(&mut self)
```

Comment:
Shouldn't we change it for `get_inactive_validators`?

Reply:
We no longer have an easy way to list the validators that are inactive, it would be a very expensive task to iterate over all validators to them list them.